sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/LogsHelper.cs

```diff
@@ -80,6 +83,45 @@ internal static string GetMessageAndSetProperties(LogRecord logRecord, IDictiona
             return message;
         }
 
+        internal static void WriteScopeInformation(LogRecord logRecord, IDictionary<string, string> properties)
+        {
+            StringBuilder builder = null;
+            int originalFormatDepth = 1;
+            logRecord.ForEachScope(ProcessScope, properties);
+
+            void ProcessScope(LogRecordScope scope, IDictionary<string, string> properties)
+            {
+                if (scope.Scope is IEnumerable<KeyValuePair<string, object>> stateDictionary)
+                {
+                    foreach (KeyValuePair<string, object> scopeItem in scope)
+                    {
+                        if (scopeItem.Key == "{OriginalFormat}")
+                        {
+                            properties.Add($"OriginalFormatScope_{originalFormatDepth++}", Convert.ToString(scope.Scope.ToString(), CultureInfo.InvariantCulture));
+                        }
+                        else if (!properties.TryGetValue(scopeItem.Key, out _))
+                        {
+                            properties.Add(scopeItem.Key, Convert.ToString(scopeItem.Value, CultureInfo.InvariantCulture));
+                        }
+                        else
+                        {
+                            AzureMonitorExporterEventSource.Log.Write($"DuplicateScopeItem{EventLevelSuffix.Warning}", $"Found duplicate scope key - {scopeItem.Key}.");
```

Comment:
its not likely an actionable item for users, as they may not control scopes. 
An alternative would be to prefix scope depth to make everything unique.

Reply:
Had the same and remove before last commit. Will add the scope depth.